### ISSUE TITLE
`tidb-server -h` output test flags 

### ISSUE THREAD

**Issue by siddontang** [pingcap/tidb#10975]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

```
./tidb-server -h
```

2. What did you expect to see?

No any -test fields in the binary. 

3. What did you see instead?

```
  -test.cpuprofile file
    	write a cpu profile to file
  -test.failfast
    	do not start new tests after the first test failure
  -test.list regexp
    	list tests, examples, and benchmarks matching regexp then exit
  -test.memprofile file
    	write an allocation profile to file
  -test.memprofilerate rate
    	set memory allocation profiling rate (see runtime.MemProfileRate)
  -test.mutexprofile string
    	write a mutex contention profile to the named file after execution
  -test.mutexprofilefraction int
    	if >= 0, calls runtime.SetMutexProfileFraction() (default 1)
  -test.outputdir dir
    	write profiles to dir
  -test.parallel n
    	run at most n tests in parallel (default 4)
  -test.run regexp
    	run only tests and examples matching regexp
  -test.short
    	run smaller test suite to save time
  -test.testlogfile file
    	write test action log to file (for use only by cmd/go)
  -test.timeout d
    	panic test binary after duration d (default 0, timeout disabled)
  -test.trace file
    	write an execution trace to file
  -test.v
    	verbose: print additional output
```

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?

```
Release Version: v3.0.0-rc.1-245-gfef22aec1
Git Commit Hash: fef22aec12dd428e2e2d143a1a9966763f332c7f
Git Branch: master
UTC Build Time: 2019-06-28 05:36:01
GoVersion: go version go1.12.5 darwin/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
```

**Comment by suzaku** [comment]
It seems that this PR was already fixed in #11207 .

**Comment by nullnotnil** [comment]
Confirming that this has been fixed:
```sql
nullnotnil@ubuntu:~/bin$ /mnt/evo970/bin/tidb-server -h
Usage of /mnt/evo970/bin/tidb-server:
  -L string
    	log level: info, debug, warn, error, fatal (default "info")
  -P string
    	tidb server port (default "4000")
  -V	print version information and exit (default false)
  -advertise-address string
    	tidb server advertise IP
  -affinity-cpus string
    	affinity cpu (cpu-no. separated by comma, e.g. 1,2,3)
  -config string
    	config file path
  -config-check
    	check config file validity and exit (default false)
  -config-strict
    	enforce config file validity (default false)
  -cors string
    	tidb server allow cors origin
  -enable-binlog
    	enable generate binlog (default false)
  -host string
    	tidb server host (default "0.0.0.0")
  -lease string
    	schema lease duration, very dangerous to change only if you know what you do (default "45s")
  -log-file string
    	log file path
  -log-slow-query string
    	slow query file path
  -metrics-addr string
    	prometheus pushgateway address, leaves it empty will disable prometheus push.
  -metrics-interval uint
    	prometheus client push interval in second, set "0" to disable prometheus push. (default 15)
  -path string
    	tidb storage path (default "/tmp/tidb")
  -plugin-dir string
    	the folder that hold plugin (default "/data/deploy/plugin")
  -plugin-load string
    	wait load plugin name(separated by comma)
  -proxy-protocol-header-timeout uint
    	proxy protocol header read timeout, unit is second. (default 5)
  -proxy-protocol-networks string
    	proxy protocol networks allowed IP or *, empty mean disable proxy protocol support
  -repair-list string
    	admin repair table list
  -repair-mode
    	enable admin repair mode (default false)
  -report-status
    	If enable status report HTTP service. (default true)
  -require-secure-transport
    	require client use secure transport
  -run-ddl
    	run ddl worker on this tidb-server (default true)
  -socket string
    	The socket file to use for connection.
  -status string
    	tidb server status port (default "10080")
  -status-host string
    	tidb server status host (default "0.0.0.0")
  -store string
    	registered store name, [tikv, mocktikv] (default "mocktikv")
  -token-limit int
    	the limit of concurrent executed sessions (default 1000)
```

I am going to close this issue now. Thanks!
